Resources/Prototypes/_Goobstation/Reagents/medicine.yml

```
@@ -112,3 +112,1266 @@
+  flavor: medicine
+  color: "#ff5936"
+  plantMetabolism:
+    - !type:PlantAdjustWeeds
+      amount: -8
+    - !type:PlantAdjustHealth
+      amount: -20
+  metabolisms:
+    Medicine:
+      metabolismRate: 1
+      effects:
+      - !type:HealthChange
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)